### PR TITLE
Added basic General Interest invite design

### LINKED ISSUE #19: Design General Interest Announcement Email

### PR DESCRIPTION
**Pull request by kberzinch**
This pull request is dependent on #45 for RSVP links and closes #19.

### CODE REVIEW

**Comment by kberzinch** [comment]
You should also definitely run this on your own machine to see how it renders - there are some non-obvious additions by default, although we can customize those as well if desired.

File: app/Mail/GeneralInterestInvite.php [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    }
+
+    /**
+     * Build the message.
+     *
+     * @return $this
+     */
+    public function build()
+    {
+        $this->from('noreply@my.robojackets.org', 'RoboJackets')
```
Comment (ryanstrat) [inline app/Mail/GeneralInterestInvite.php]: Can we set this info in a config file instead?

File: app/Mail/GeneralInterestInvite.php [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+             ->subject('RoboJackets General Interest Meeting - RSVP Requested')
+             ->markdown('mail.generalinterest.invite');
+
+        $this->withSwiftMessage(function ($message) {
+            $message->getHeaders()
+                    ->addTextHeader('Reply-To', 'RoboJackets Support <support@robojackets.org>');
```
Comment (ryanstrat) [inline app/Mail/GeneralInterestInvite.php]: Sending from a no-reply address seems weird to me if the reply-to is set differently. I don't know what the standards for this are exactly, though

File: app/Mail/GeneralInterestInvite.php (hunk 1 above)
Comment (kberzinch) [inline app/Mail/GeneralInterestInvite.php]: There's a default configuration in .env that is correct for most mail, but I want to send GI invites as "RoboJackets" not "MyRoboJackets".

File: app/Mail/GeneralInterestInvite.php (hunk 2 above)
Comment (kberzinch) [inline app/Mail/GeneralInterestInvite.php]: We could leave off the Reply-To. As a general rule, nobody should be replying to this email, but if they do I want it to go to the admin mailbox, hence the setup.

Alternatively we can send from support@my, which also goes to the admin mailbox.

**Comment by ryanstrat** [comment]
Have we dealt with the headers / footers that needed to be removed yet?

**Pull request merged**